Makefile.docker

```diff
@@ -15,7 +15,7 @@ all: docker_build docker_push
 
 docker_build:
 	# Build Docker image ...
-	docker build -t strimzi/$(PROJECT_NAME):latest $(DOCKERFILE_DIR)
+	docker build -q -t strimzi/$(PROJECT_NAME):latest $(DOCKERFILE_DIR)
```

Comment:
I think this will make it a bit tricky to see why the Docker image build crashed (which also happens from time to time because of YUM timeouts). But lets give it a try.